solidity/contracts/libraries/staking/MinimumStakeSchedule.sol

```diff
@@ -0,0 +1,19 @@
+pragma solidity 0.5.17;
+
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+
+library MinimumStakeSchedule {
```

Comment:
Would it be worth adding docs to this library?

Reply:
Done 👍